plugins/process.py

```diff
@@ -27,7 +27,6 @@ def generate_gerber(self, temp_dir):
         plot_options.SetScale(1)
         plot_options.SetMirror(False)
         plot_options.SetUseGerberAttributes(True)
-        plot_options.SetExcludeEdgeLayer(True)
```

Comment:
Could you conditionally ignore this line based on the software version?

Reply:
Good idea, just did that to both places I had to touch.

Reply:
Thanks!